botocore/httpsession.py

```diff
@@ -81,6 +82,13 @@ def create_urllib3_context(ssl_version=None, cert_reqs=None,
         # We do our own verification, including fingerprints and alternative
         # hostnames. So disable it here
         context.check_hostname = False
+
+    # OpenSSL 1.1.1 keylog file
+    if hasattr(context, 'keylog_filename'):
+        keylogfile = os.environ.get('SSLKEYLOGFILE')
+        if keylogfile and not sys.flags.ignore_environment:
```

Comment:
Can we get some validation that `sys.flags.ignore_environment` is supported on all Python versions?

Reply:
According to the docs, sys.flags.ignore_environment is present since Python 2.7: https://docs.python.org/2.7/library/sys.html

Reply:
I've updated the comments, and added a backlink to upstream in the commit message

Reply:
Upon further inspection, can we get testing to show that `sys.flags.ignore_enviorment` properly opts out of setting the keylog file path in the `SSLContext`. Requesting this since it is not part of the upstream implementation. Thanks!

Reply:
I'm not sure I understand your request...
Do you want me to add a test to verify that the if-statement on line 90 actually works for the `ignore_environment`-part?

Reply:
After running through the possible outcomes if something were to change in the future, I think everything looks good as is. I wanted to see if we could do anything more to make this robust but your question is fair. I will approve and get this merged.